### ISSUE TITLE
Overkilling voice instructions for turn destinations with more than one entry

### ISSUE THREAD

**Issue by WizoneDE** [osmandapp/OsmAnd#16226]
### Description
when getting a turn instruction with more than one turn destination set the app is currently read aloud ALL of the destinations.
voice instruction should only read aloud the FIRST of the destination informations and trim all the others from voice instruction

Example:
https://osmand.net/map/?start=50.449211%2C7.844131&end=50.451855%2C7.850118&mode=car#18/50.45069/7.84714

destination=Ruppach-Goldhausen;Heiligenroth;Heiligenroth
destination:colour=;;white
destination:symbol=;;industrial

current voice instruction:
in xxx meter Turn slightly right Ruppach-Goldhausen Heiligenroth Heiligenroth 

expacted voice Instruction:
in xxx meter Turn slightly right Ruppach-Goldhausen 

### How to reproduce?
drive a route with more than one destination set on segment

### Your Environment
OsmAnd Version: 4.3.3
iOS version: 16.2
Device model: 14 Pro

**Maps used (online or offline):**   
online and offline with 1h update

**Comment by Zirochkabila** [comment]
@WizoneDE You're right, the voice prompts sound like "in xxx meter Turn slightly right Ruppach-Goldhausen Heiligenroth Heiligenroth", but the point is that OsmAnd uses the information stored in OSM - accordingly, the destination is displayed as Ruppach-Goldhausen;Heiligenroth;Heiligenroth (you can see it in the screenshot and here). If you change the destination to Ruppach-Goldhausen, the voice guidance will work differently.

[OSM](https://www.openstreetmap.org/way/23879375#map=19/50.45175/7.84759)
<details><summary>Ruppach-Goldhausen Heiligenroth Heiligenroth</summary>
<img width="800" alt="Screenshot 2023-01-31 at 11 19 41" src="https://user-images.githubusercontent.com/104760013/218300007-08e084a3-0d30-4e6a-b479-7ac157e310b2.png"">
</details>

**Comment by WizoneDE** [comment]
Yea but then I would corrup correct data in osm? Why should I do this? Check the osm definition for destination, ref and icons. The 2 Heiligenroth is linked to an industrial icon. 

For OsmAnd:
Just cut everything behind the first ; in a destination string and both is working,

voice prompt is just giving the first name of the destination sign and the osm data doesn’t need to be corrupted for OsmAnd

**Comment by pebogufi** [comment]
For me it is ok as it currently works

**Comment by vshcherb** [comment]
Looks like data is ok and voice pronounces all names 
![Screenshot 2023-03-01 at 18 27 25](https://user-images.githubusercontent.com/1042025/222201354-0a77a148-cbcf-4935-9330-fb44850d3cb7.png)
